### PR TITLE
Add tags to reference github prs and issues

### PR DESCRIPTION
**Pull request by Soulofdestiny**
As mentioned in Issue #604,
for now it's only referencing pull requests to os-autoinst-distri-opensuse

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
tested locally

**Comment by aaannz** [comment]
On github you can address pull requests or issues. This only handle pull requests. What about `ghp#` and `ghi#`?

File: t/ui/15-comments.t [hunk 1]
```diff
@@ -69,7 +69,7 @@ $driver->find_element('textarea', 'css')->send_keys('
     foo@bar foo#bar
     <a href="https://openqa.example.com/foo/bar">https://openqa.example.com/foo/bar</a>: http://localhost:9562
     https://openqa.example.com/tests/181148 (reference http://localhost/foo/bar )
-    bsc#1234 boo#2345 poo#3456 t#4567
+    bsc#1234 boo#2345 poo#3456 t#4567 gh#6789
     t#5678/modules/welcome/steps/1'
```
Comment (aaannz) [inline t/ui/15-comments.t]: wrong line, you need to put it after L73 or switch tests on L90-91

**Comment by Soulofdestiny** [comment]
Thanks @aaannz  - was in a little rush :)  
I will change this - and I like the idea of ghp and ghi

**Comment by aaannz** [comment]
You seem not to address https://github.com/os-autoinst/openQA/issues/604 - links are still openSUSE tests or os-autoinst related. Especially for issues you can have os-autoinst or openQA. I think combinations like `ghi#os-autoinst/\d+` and `ghi#openqa/\d+` and possible generic one `ghi#os-autoinst/openqa/\d+` should be supported. (And similar combinations for `ghp#`)
Then, I don't expect you to do it, question rises if we should move these translations to generic config file instead having them in the code.

**Comment by okurz** [comment]
what about `gh:<username>/<repo>#\d+`, e.g. "gh:os-autoinst:openQA#666"?

**Comment by coolo** [comment]
how about just using full links instead of such cryptic steno?

**Comment by coolo** [comment]
I still don't like that - just use a link if you want to link github repos

**Pull request closed without merging**